packages/web-example/src/prod.ts

```diff
@@ -87,29 +86,28 @@ const runTest = async (testConfig: TestConfig) => {
     console.log("errr", err);
     throw new Error(`${err} sss_error`);
   });
-
+  
   console.log("initializing tkey");
   console.log("fetching sss pub keys using 'fetchPostboxKeyAndSig'");
 
-  await tb.initialize({ useTSS: true, factorPub, deviceTSSShare, deviceTSSIndex }).catch((err) => {
+  await tb.initialize({ useTSS: true, factorPub, deviceTSSShare, deviceTSSIndex }).catch((err: unknown)=>{
     throw new Error(`${err} tkey_error`);
   });
   console.log("syncing tkey metadata");
 
-  await tb.syncLocalMetadataTransitions().catch((err) => {
+  await tb.syncLocalMetadataTransitions().catch((err: unknown)=>{
     throw new Error(`meta-data error:${err}`);
```

Comment:
unrelated, but could add a space here
```suggestion
    throw new Error(`meta-data error: ${err}`);
```